src/makefiles/linux_clapack.mk

```diff
@@ -13,6 +13,10 @@ ifndef OPENFSTLIBS
 $(error OPENFSTLIBS not defined.)
 endif
 
+CLAPACKLIBS = $(CLAPACKROOT)/CLAPACK-3.2.1/lapack.a $(CLAPACKROOT)/CLAPACK-3.2.1/libcblaswr.a \
+	      $(CLAPACKROOT)/CBLAS/lib/cblas.a \
+	      $(CLAPACKROOT)/f2c_BLAS-3.8.0/blas.a $(CLAPACKROOT)/libf2c/libf2c.a
+
```

Comment:
Please remove these hard coded libs.

These will break builds on many linux systems.

If you need to modify the ld flags.

Please use
```
LDFLAGS='/path/to/your/static/libs.a' ./configure
```
or
```
LDFLAGS='-lcblas -llapack' ./configure
```
when compiling the packages.

Reply:
Thanks!
We very rarely compile with CLAPACK.  We'd need to test any changes to the Makefiles.  If you have time to do this and make a PR then great, but this isn't a high priority right now.